tests/test_compiler.py

```diff
@@ -169,3 +169,40 @@ def test_dict():
         {'foo-value': env['foo'](1), 'bar-value': env['bar'](2), }
         """
     )
+
+
+def test_generic_none():
+    check_compiles(
+        None,
+        "None"
+    )
+
+
+def test_generic_bool():
+    check_compiles(
+        True,
+        "True"
+    )
+
+
+def test_generic_py2_long():
+    check_compiles(
+        2 ** 64,
+        "18446744073709551616L"
```

Comment:
I believe that here is an error, in Py3 there are no trailing `L` for int values. The reason why tests are not failing is because astor library doesn't support Python 3.5 and higher, and Hiku doesn't support Python 3.4, so tests in Py3 are checking only that expression were successfully compiled.